src/msix/PAL/XML/xerces-c/XmlObject.cpp

```diff
@@ -402,6 +533,124 @@ class XercesDom final : public ComClass<XercesDom, IXmlDom>
     }
 
 protected:
+
+    std::unique_ptr<MemBufInputSource> StripIgnorableNamespaces(const InputSource& source, const NamespaceManager& namespaces)

```

Comment:
>StripIgnorableNamespaces [](start = 39, length = 24)

Would be nice if this could live above the PAL, but the cost of projecting everything that would be needed seems high.